src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -442,11 +442,20 @@ RCLConsensus::doAccept(
 
     if (validating_ && !consensusFail)
     {
-        validate(sharedLCL, proposing);
-        JLOG(j_.info()) << "CNF Val " << newLCLHash;
+        validate (sharedLCL, proposing);
+        JLOG (j_.info()) << "CNF Val " << newLCLHash;
+    }
+    else if (haveCorrectLCL && !consensusFail && valPublic_.size())
+    {
+        validating_ = true;
```

Comment:
I'm concerned this is not fixing the underlying bug, which seems to be in how [`validated_`](https://github.com/mtrippled/rippled/blob/4f1a4a971ef2ee61fa6e574331f365919f3f5420/src/ripple/app/consensus/RCLConsensus.cpp#L925-L930) is being set at the start of a round, or if somehow `isNeedNetworkLedger` is never getting cleared properly.  

Is it the case that once the node issues the boostrapping validation, it never does so again and only issues normal validations?

Given that this fixes the problem you've observed, I'd like a little time to first see if I can implicate one of the above alternate causes.  Those might still be causing other trouble that we just haven't yet observed.